sdcm/nemesis.py

```diff
@@ -2601,9 +2601,9 @@ def disrupt(self):
                    GeminiChaosMonkey, NetworkMonkey]
 
 
-class CorruptThenScrubMonkey(Nemesis):
+class DisableCorruptThenScrubMonkey(Nemesis):
```

Comment:
comment this out with todo comment linking to the task https://trello.com/c/vwedwZK2/1881-corrupt-the-scrub-fails

Reply:
this is needed since some nemesis groups are generated automatically based on all the classes that inherit from Nemesis

Reply:
Oh, I thought that adding "Disable" at the beginning will dodge such cases.
In any case, done and done